enterprise/backend/src/metabase_enterprise/sso/integrations/saml.clj

```diff
@@ -135,7 +136,7 @@
   [req]
   (let [redirect (get-in req [:params :redirect])
         origin (get-in req [:headers "origin"])
-        embedding-sdk-header? (sso-utils/is-embedding-sdk-header? req)]
+        embedding-sdk-header? (embed.util/has-react-sdk-header? req)]
```

Comment:
On master, I use `embed.util/is-modular-embedding-request?`. But since we don't have embedded analytics JS on 55, I only have a single function `embed.util/has-embedding-sdk-header?` in the `embedding.util` now.